components/ProgressBar/OurProgressBar.jsx

```diff
@@ -0,0 +1,5 @@
+import { Progress } from "@chakra-ui/react"
+
+export default function OurProgressBar({progressValue}) {
```

Comment:
The `OurProgressBar` component currently takes a `progressValue` that is the 'percentage of shading' for progress bar. I think could make it take a simpler value by just providing the page number as an argument, and calculating the correct percentage in this component. 
We will need to know the total number of questions to perform such a calculation; so we could consider providing that value in [here](https://github.com/CodeforAustralia/council-emissions-calculator-spike/blob/main/utils/constants.js) and exporting it for use in this page.
not a biggie at this point (there are only 5 questions atm!); just a suggestion for future improvement 

other suggestions:
- add some 'bottom padding' between the progress bar and the title?
- add some text in the progress bar to indicate the progression? eg. '1 of 5', or maybe just a percentage at least?
- suggest a different component name for `OurProgressBar` ; something more descriptive/objective like `FormProgressBar`, or `CompletionStatusBar`?

Reply:
Thanks Qing @hqtan 
- Yes, I agree about the calculation, but yes, let's see how many questions come in, because changing it could be more trouble than it's worth without that side being clear (how many more questions we can expect - if any). If they don't change then we might be getting fancy with no real cause.
- Bottom padding issue: I would tend not to want to do that. The reason is that in my view the layout of the page should be addressed as a whole, and each component should be kept 'clean' so that the whole layout can be taken into consideration without needing to then worry about each element having various amounts of padding and margin. My view is that Flexbox should be sent to do the work to the page as a first step, and then specific elements might be nudged later if needed.
- Happy to add some text to the bar - it's a good suggestion - but again this might come back to design. If there are not going to be, say, at least 5 more questions, then although it's not as 'nice', I think I would recommend the SVG option. The amount you gain by programmability is not really worth it unless there are many questions and they can change frequently. That said, happy to be told what to do, and if that is use this bar, and have some text in there, all good. It's the same issue as the first one. I can change the colour easily too.
- Yep, can change the name of the progress bar. I don't have a strong opinion, so you can choose the name, and I can change it.